### PR TITLE
Render with b64 improvements

### PR DESCRIPTION
**Pull request by sobil**
WHAT

Updates packages, fixes tests

WHY

Small improvment in size of payload and larger increase in speed

### CODE REVIEW

**Comment by sobil** [comment]
Results from performance test:

> Total time taken to render props 25 times: 2.8778889179229736
> Max: 0.06415295600891113
> Min: 0.05672502517700195
> Mean: 0.057557778358459474
> Median: 0.05696702003479004
> 
> Total time taken to render props b64 25 times: 0.3842003345489502
> Max: 0.008542060852050781
> Min: 0.0075130462646484375
> Mean: 0.007684006690979004
> Median: 0.00763702392578125

File: tests/test_functionality.py [hunk 1]
```diff
@@ -83,12 +83,33 @@ def test_can_get_a_components_serialized_props(self):
...
+        self.assertEqual(len(component.render_props()), 98)
+        self.assertEqual(len(component.render_props_b64()), 80)
+
     def test_can_serialize_datetime_values_in_props(self):
         component = render_component(
             PATH_TO_HELLO_WORLD_COMPONENT_JSX,
             props={
                 'name': 'world!',
-                'datetime': datetime.datetime(2015, 1, 2, 3, 4, 5, tzinfo=timezone.utc),
+                'datetime': datetime.datetime(2015, 1, 2, 3, 4, 5, tzinfo= timezone.utc),
```
Comment (kertonandrew) [inline tests/test_functionality.py]:
```suggestion
                'datetime': datetime.datetime(2015, 1, 2, 3, 4, 5, tzinfo=timezone.utc),
```

**Review by kertonandrew — COMMENTED** [review]
1 change

File: react_render/django/render.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ def render_props(self):
 
     def render_props_b64(self):
         if self.props:
+            self.json_encoder = json.JSONEncoder(separators=(',', ':')).encode
             encoded = base64.b64encode(self.json_encoder(self.props).encode("utf-8")).decode('utf-8')
```
Comment (akshayw39) [inline react_render/django/render.py:44]: This has an issue in the current form it does not work 
work around:
`json.dumps(self.props, cls=self.json_encoder.__class__)`

File: react_render/django/render.py [hunk 3]
```diff
@@ -41,6 +41,7 @@ def render_props(self):
 
     def render_props_b64(self):
         if self.props:
+            self.json_encoder = json.JSONEncoder(separators=(',', ':')).encode
```
Comment (akshayw39) [inline react_render/django/render.py]: this needs to be DjangoJSONEncoder which is passed. Which formats decimals, datetime etc

Comment (sobil) [reply react_render/django/render.py]: Can you write me a test to demonstrate?

**Pull request closed without merging**